src/luxonis_ml/loader/loader.py

```diff
@@ -1,18 +1,30 @@
-from pathlib import Path
 import cv2
 import numpy as np
 import torch
+import random
+import warnings
+from pathlib import Path
 from fiftyone import ViewField as F
+from enum import Enum
+
+
+class LabelType(str, Enum):
```

Comment:
I like `LabelType`. I think we could actually define it in the `data` module and use it there as well. I think it will be a new feature but if we agree I'll note it down.

Reply:
Yeah I was thinking about using it in `data` module as well. But I wasn't sure how to then still keep modules separate because if we define it in `data` then we would need to import `data` in `loader`. Should there maybe be a separate file with constants outside of modules with shared types?

Reply:
Oh yeah maybe we can move all the enums to some enums folder? Or something like constants. I saw @tersekmatija did this in some project.

Reply:
In `models` I have them under utils/constants.py but that is a single module. So yeah here we could do something similar, just outside of specific module.